lib/services/github_api.dart

```diff
@@ -33,13 +33,16 @@ class GithubAPI {
     });
   }
 
-  Future<Map<String, dynamic>?> getLatestRelease(
-    String repoName,
-  ) async {
+  Future<Map<String, dynamic>?> getLatestRelease(String repoName) async {
+    final String target =
+        _managerAPI.usePrereleases() ? '?per_page=1' : '/latest';
     try {
       final response = await _dioGetSynchronously(
-        '/repos/$repoName/releases/latest',
+        '/repos/$repoName/releases$target',
       );
+      if (_managerAPI.usePrereleases()) {
+        return response.data.first;
+      }
       return response.data;
```

Comment:
@kitadai31 would this work:
```suggestion
        return response.data.first;
```

Reply:
No that won't work. `/latest` provides Map, the other one provides List. So, need to add the condition.